arviz/rcparams.py

```diff
@@ -35,7 +35,10 @@ def validate_choice(value):
             raise ValueError("Could not convert to {}".format(typeof.__name__))
         if isinstance(value, str):
             value = value.lower()
+
         if value in accepted_values:
+            # Convert value to python boolean if string matches
+            value = {"true": True, "false": False}.get(value, value)
```

Comment:
@OriolAbril Is this a good idea?

Reply:
I think this will do the trick, but I find it quite confusing. For example, if someone where to use:

```
with az.rc_context({"plot.bokeh.show": True}):
    az.plot_xyz()
```

It would convert `True` to a string, check whether it was in accepted values and then convert it back to a boolean. Moreover, even though the `typeof` would be string and the accepted values would be strings too, the return value would be of boolean type.

Reply:
What if user uses "True" or "False"

Reply:
It would work because RcParams lowercases the params before checking.

@OriolAbril I'd agree it'd be confusing if someone checks under the hood. I'll make another proposal

Reply:
Actually I'm not sure what do here. Because in RcParams file the input will be a string, but in context manager the input will be string but it needs to be converted to a bool. But if it comes from context manager it'll be a bool and need to ultimately return a bool.

Since `validate_choice` doesn't know the source not sure what to do that is least confusing

Reply:
My proposal was to create a new `_validate_boolean` that returns `True` (as boolean) if value is `True` or `"true"`, returns `False` (as boolean) if value is `False` or `"false"` and raises an error otherwise.